screens/metricTank

```diff
@@ -20,9 +20,7 @@ cd /go/src/github.com/raintank/raintank-metric/metric_tank
 	--listen :6063 \
 	--primary-node \
 	--elastic-addr elasticsearch:9200 \
-	--redis-addr redis:6379 \
 	--elastic-warmup-pct 100 \
-	--cassandra-addrs cassandra &>> /var/log/raintank/nsq_metrics_tank.log &
-tail -f /var/log/raintank/nsq_metrics_tank.log 
-
+	--cassandra-addrs cassandra \
+|& tee /var/log/raintank/nsq_metrics_tank.log 
```

Comment:
hmm, i disagree with this. I typically only want to kill the service and restart it.

If i want to do anything else interactive i just start up a new shell on the container. eg. `docker exec -t -i raintankdocker_grafana_1 bash`